atomicapp/cli/main.py

```diff
@@ -399,9 +411,12 @@ def run(self):
         if args.mode:
             args.action = args.mode     # Allow mode to override 'action'
         cmdline.insert(0, args.action)  # Place 'action' at front
+        # Setup logging (now with arguments from cmdline)
+        Logging.setup_logging(args.verbose, args.quiet, args.logging_output)
         logger.info("Action/Mode Selected is: %s" % args.action)
 
         # Finally, parse args and give error if necessary
+        logger.debug("Final parsed cmdline: {}".format(' '.join(cmdline)))
         args = self.parser.parse_args(cmdline)
```

Comment:
I think I see what you are saying now. You are saying that `self.parser.parse_args()` needs to be moved above the `setup_logging()` call. I don't think this is true because the call to ` self.parser.parse_known_args()` should have captured `--verbose` from the cmdline. I'll run some tests to see if it changes things.